dataland-frontend/src/components/forms/parts/kpiSelection/AssuranceFormField.vue

```diff
@@ -122,7 +117,7 @@ export default defineComponent({
     };
   },
   mounted() {
-    setTimeout(() => (this.isMounted = true));
+    void nextTick(() => (this.isMounted = true));
```

Comment:
sonar meckert meistens über den void operator, folgendes kann helfen: nextTick(()=> (this.isMounted = true)).catch((error)=> console.error(error));

Reply:
Das scheint aber ein False-Positive von Sonar zu sein. Der Void operator ist da vollkommen valide eingesetzt.